client/features/auth/authSlice.js

```diff
@@ -1,34 +1,43 @@
-import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
-import axios from 'axios';
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
 
 /*
   CONSTANT VARIABLES
 */
-const TOKEN = 'token';
+const TOKEN = "token";
 
 /*
   THUNKS
 */
-export const me = createAsyncThunk('auth/me', async () => {
+export const me = createAsyncThunk("auth/me", async () => {
   try {
     const token = window.localStorage.getItem(TOKEN);
+    console.log("token:", token);
 
+    // if (token = undefined) {
+    //   console.log('token undefined BUT THIS IS WORKING')
+    // }
     if (token) {
-      const res = await axios.get('/auth/me', {
+
+      // const via = await axios.get("/visitors");
+      // console.log('vis,' via);
+      console.log('if');
+      const res = await axios.get("/auth/me", {
         headers: {
```

Comment:
could you delete the commented code and take out the console.logs in this thunk?